eos/effects/fighterabilityevasivemaneuvers.py

```diff
@@ -0,0 +1,24 @@
+# Not used by any item
+"""
+Since fighter abilities do not have any sort of item entity in the EVE database, we must derive the abilities from the
+effects, and thus this effect file contains some custom information useful only to fighters.
+"""
+# User-friendly name for the ability
+displayName = "Evasive Maneuvers"
+
+prefix = "fighterAbilityEvasiveManeuvers"
+
+# Is ability applied to the fighter squad as a whole, or per fighter?
+grouped = True
+
+type = "active"
+runTime = "late"
+def handler(fit, module, context):
```

Comment:
Yeah, I've been calling it `src` as it's whatever item the effect is coming from. I'm not worried about that though.